www-static/index.html

```diff
@@ -2,15 +2,44 @@
 
 <html lang="en">
 <head>
-    <title>Code City Static Files</title>
-    <style>
-body {
-  font-family: sans-serif;
-}
-    </style>
+  <title>Code City</title>
+  <style>
+    body {
+      font-family: 'Roboto Mono', monospace;
+      text-align: center;
+    }
+    h1 {
+      font-size: 40pt;
+      font-weight: 100;
+    }
+    h1>svg {
+      vertical-align: text-bottom;
+    }
+    #tagline {
+      font-style: italic;
+      margin: 2em;
+    }
+    iframe {
+      height: 50px;
+      width: 100px;
+      border: none;
+      display: block;
+      margin: 0 auto;
+    }
+  </style>
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet">
+  <link href="/favicon.ico" rel="shortcut icon">
 </head>
 <body>
-    <h1>Code City Static Files</h1>
-    <p>This directory contains static files which are served to the web using Nginx.</p>
+  <h1>
+    <svg xmlns="http://www.w3.org/2000/svg" width="95px" height="100px">
+      <path
+         d="m 1,90 0,-25 10,0 0,-8 9,0 0,-5 8,0 0,5 2,0 0,33 3,0 0,-13 2,0 0,-7 2,0 0,-32 1,0 0,-7 1,0 0,-6 2,0 0,-1 1,0 0,-12 1,0 0,-12 0,12 1,0 0,12 1,0 0,1 2,0 0,6 1,0 0,7 1,0 0,32 2,0 0,7 2,0 0,13 4,0 0,-32 5,0 0,-5 6,0 0,5 3,0 0,32 5,0 0,-42 3,0 0,-9 7,0 0,-2 5,0 0,2 2,0 0,51"
+         style="fill: none; stroke: #000; stroke-width: 1.5;" />
+    </svg>
+    Code City
+  </h1>
+  <p id="tagline">A community of young programmers.</p>
```

Comment:
How about something less age-specific than "young"?  Perhaps "budding" or "aspiring"?

Reply:
"Budding" will get us into trouble with HR.  "Aspiring" implies that they aren't programmers.  "Inquisitive"?

Reply:
"Budding: (of a person) beginning and showing signs of promise in a particular sphere."  Not sure what the HR problem is.  I think that pretty much sums up exactly our target group.

Reply:
(But "inquisitive" is O.K. too.)

Reply:
That's the third definition.  The second is the problematic one.